proposals/mpi-operator-proposal.md

```diff
@@ -12,7 +12,9 @@ multi-node training using TensorFlow; the TensorFlow team is also working on an
 [implementation](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/distribute).
```

Comment:
Could we add a link to http://mpitutorial.com/tutorials/mpi-reduce-and-allreduce/ for the introduction to allreduce here:

```diff
- with each other directly via the MPI `allreduce` primitive
+ with each other directly via the MPI [`allreduce`](http://mpitutorial.com/tutorials/mpi-reduce-and-allreduce/) primitive
```

Reply:
Sure.